docs/guides/server-side-rendering.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Server side rendering (SSR)
+category: Guides
+order: 1
+---
+
+# SSR with Next.js
+
+This document will demonstrate how to achieve Server Side Rendering (SSR) with [Next.js](https://nextjs.org/) using InstUI components.
+
+> This document assumes that you already have a Next.js application in place.
+
+#### Initial steps
+
+- install InstUI related dependencies:
+
+```sh
+yarn add @instructure/emotion @instructure/ui-react-utils
+```
+
+> Note!
+> You should not install the InstUI meta package(`@instructure/ui`) since it has components that are not yet Server Side Renderable out of the box.
+> The `@instructure/ui-code-editor` currently relies on browser specific APIs and requires that your choice of bundler/transpiler to touch the source code of this component, so effectively it is not in a usable state for out of box usage with Next.js.
+> You should install only the packages you need!
```

Comment:
Didn't we decided to fix CodeEditor (and accept that it might break Jest tests)?

Reply:
Yes, I removed it